modules/compute/eks/stack.py

```diff
@@ -681,6 +682,10 @@ def _create_vpc_cni_chart(
         """
         Creates the VPC CNI Helm chart.
         """
+
+        ECR_DOMAIN = fetch_ecr_domain(self.region)
```

Comment:
Does CDK provide the [`AWS::URLSuffix` that CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html#cfn-pseudo-param-partition) allows?

Reply:
Thanks for spotting it. Let me test and get back. Yes, CDK provides [domain](https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk/Stack.html#aws_cdk.Stack.url_suffix) support

Reply:
It worked :)